package.json

```diff
@@ -95,5 +96,8 @@
     "vscode-languageserver-protocol": "^3.14.1",
     "ws": "^7.2.1",
     "yarn": "^1.22.0"
+  },
+  "resolutions": {
+    "vscode-languageserver-protocol": "3.14.1"
```

Comment:
Not sure why this is needed, could you explain?

Reply:
I couldn't get this to resolve at the time. It would always pick a non-compatible version. Seems fine now after other dependency updates.

https://github.com/sourcegraph/sourcegraph-typescript/pull/340